src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -200,8 +201,21 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
 
 export const ArtistAuctionResultsRefetchContainer = createRefetchContainer(
   (props: AuctionResultsProps) => {
+    const {
+      startAt,
+      endAt,
+    } = props.artist?.auctionResultsConnection?.createdYearRange
+    // TODO: Remove once diffusion data is cleaned up
+    const { birthday } = props.artist
+    const birthYear = DateTime.fromJSDate(new Date(birthday)).year
     return (
-      <AuctionResultsFilterContextProvider>
+      <AuctionResultsFilterContextProvider
+        filters={{
+          earliestCreatedYear:
+            birthday && birthYear > startAt ? birthYear : startAt,
+          latestCreatedYear: endAt,
+        }}
+      >
```

Comment:
@jpotts244, I'm doing a _bit_ of massaging here just to ensure we're not showing a work with a year created that's earlier than the artist's birth year. Once we're able to clean up the data a bit in diffusion to ensure this isn't really much of an issue we can nix this.

Reply:
🙏 thanks for the coverage here! hoping the additional constraints in the new ArtistMatch job will help with cases like these

Reply:
It may be worth creating a task to actually do this logic in diffusion (on route that actually returns the range).

Reply:
I dig it -- might be fun to write it together if you have some time tomorrow. Would like to get your ideas on endpoint design. We could probably add some fun caching to that endpoint too!